Add sections on compensation and review process to translation FAQs (#5234)
ethereum/ethereum-org-website: src/content/contributing/translation-program/faq/index.md

If you're new to the Translation Program and are hesitant to jump in, here are some FAQs that can help you begin. Use this guide to find answers to the most common queries.

> ## Can I get compensated for translating ethereum.org? {#compensation}

Reviewer: At second glance, I don't think this should be the top question.

Author: You're probably right. I added it here since it's by far the most common question we get asked. Feel free to move it wherever you think would be more appropriate.

Reviewer 2: @minimalsm Where do you think we should place it? It wouldn't really make sense in the middle given the rest of the questions are all about the actual translation process, and the alternative is at the bottom or on another page. If we want to keep it on this page I think it's fine at the top for now, can always reevaluate and move it in the future.

> Ethereum.org is an open source website, which means that anyone can get involved and contribute.

Reviewer: ```suggestion
Ethereum.org is an open-source website, which means that anyone can get involved and contribute.
```

> The ethereum.org Translation Program is an extention of that and is organized with a similar philosophy in mind.

Reviewer: ```suggestion
The ethereum.org Translation Program is an extension of that and is organized with a similar philosophy in mind.
```

> The aim of the Translation Program is to localize the website and make Ethereum content accessible to everyone, regardless of their language, but also to allow anyone who is bilingual to get involved with the Ethereum ecosystem and contribute in an accessible way.

Reviewer: ```suggestion
The goal of the Translation Program is to make Ethereum content accessible to everyone, regardless of the languages they speak. It also allows any bilingual person to get involved with the Ethereum ecosystem and contribute in an accessible way.
```

> For this reason, the Translation Program is open and voluntary, and participation is not subject to compensation. If we decided to pay translators for their work on the website, we would only be able to accept professional translators with sufficient experience, and add other restrictions to who can participate in the Translation Program. This would make the Translation Program exclusionary and prevent us from reaching the goals, outlined above, specifically to allow everyone to participate and get involved with the ecosystem.

Reviewer: ```suggestion
For this reason, the Translation Program is open and voluntary, and participation is not subject to compensation. If we were to compensate translators for the number of words they translate, we could only invite those with sufficient translation experience (professional translators) to join the translation program. This would make the Translation Program exclusionary and prevent us from reaching the outlined goals, specifically: allowing everyone to participate and get involved with the ecosystem.
```

> That being said, we try to acknowledge our translators in different ways, like [offering POAPs](/contributing/translation-program/acknowledgements/#poap) and a [translator certificate](/contributing/translation-program/acknowledgements/#certificate), as well as organizing the [Translation Leaderboards](/contributing/translation-program/acknowledgements/) and [listing all of our translators on the site](/contributing/translation-program/contributors/).

Reviewer: ```suggestion
We make every effort to enable our contributors to succeed in the Ethereum ecosystem; many non-monteary incentives are in place such as: [offering POAPs](/contributing/translation-program/acknowledgements/#poap) and a [translator certificate](/contributing/translation-program/acknowledgements/#certificate), as well as organizing the [Translation Leaderboards](/contributing/translation-program/acknowledgements/) and [listing all of our translators on the site](/contributing/translation-program/contributors/).
```

We recommend you leave names of brands, companies, and personnel untranslated as a translation might cause unnecessary confusion and SEO difficulties.

## How does the review process work? {#review-process}

> To ensure a certain level of quality and consistency in our translations, we work with [Acolad](https://www.acolad.com/) as our language service provider.

Reviewer: ```suggestion
To ensure a certain level of quality and consistency in our translations, we work with [Acolad](https://www.acolad.com/), one of the largest language service providers globally. They work with 20,000 professional linguists and can provide professional reviewers in any language.
```

Acolad is one of the largest language service providers in the world, working with 20,000 professional linguists. This means that they can provide professional reviewers for every language and type of content we need.

> The review process is fairly straightforward - once a certain [content bucket](/contributing/translation-program/content-buckets) is 100% translated, we order a review for this specific bucket.

Reviewer: ```suggestion
The review process is straightforward; once a certain [content bucket](/contributing/translation-program/content-buckets) is 100% translated, we order a review for that content bucket. The review process takes place directly in Crowdin. Once the review is complete, we update the website with the translated content.
```

> The review process takes place directly in Crowdin, and once it is complete, we add the translated content to the website.

Reviewer: Merged into paragraph above

```suggestion
```
